plugins/ag-charts-generate-example-files/src/executors/generate/generator/constants.ts

```diff
@@ -11,3 +11,24 @@ export const SOURCE_ENTRY_FILE_NAME = 'main.ts';
  * The main angular file name that is generated from the source entry file
  */
 export const ANGULAR_GENERATED_MAIN_FILE_NAME = 'app.component.ts';
+
+export const INTERNAL_FRAMEWORK_DEPENDENCIES: Partial<Record<InternalFramework, Record<string, string>>> = {
+    angular: {
+        '@angular/core': '^14',
+        '@angular/common': '^14',
+        '@angular/forms': '^14',
+        '@angular/platform-browser': '^14',
```

Comment:
Can they use 17?

```suggestion
        '@angular/core': '^17.0.0',
        '@angular/common': '^17.0.0',
        '@angular/forms': '^17.0.0',
        '@angular/platform-browser': '^17.0.0',
```

Reply:
It matches our system js config, which points to `14.2.6`

https://github.com/ag-grid/ag-charts/blob/latest/packages/ag-charts-website/public/example-runner/charts-angular-boilerplate/systemjs.config.dev.js#L2

I'm hesitant to change it here, as I'm not sure what our support level for angular is. Can do this in another iteration